### ISSUE TITLE
'OTSubscriberView' only refers to a type, but is being used as a value here

### ISSUE THREAD

**Issue by pedrobslisboa** [opentok/opentok-react-native#503]
# Bug Report

**Current behavior**

- Problem with TS and OTSubscriberView

**Steps to reproduce**

- Install v0.17.0 and import OTSubscriberView anywhere

**What is the current *bug* behavior?**

- TS is poping this error: 'OTSubscriberView' only refers to a type, but is being used as a value here.

**What is the expected *correct* behavior?**

- It should not pop this error since OTSubscriberView is a component

**Relevant logs and/or screenshots**

![Screenshot 2021-05-14 at 09 13 15](https://user-images.githubusercontent.com/35539594/118269418-17f52280-b495-11eb-8689-dbbcb6ace794.png)

**Comment by abdulajet** [comment]
I believe it's `OTSubscriber` you need? https://github.com/opentok/opentok-react-native-samples/blob/c06be49c54fe84715a672aae8e4cff1d118351e8/BasicVideoChat/App.js#L18

**Comment by pedrobslisboa** [comment]
@abdulajet actually I want more than one view at OTSubscriber, so I need OTSubscriberView as a child of OTSubscriber.

OTSubscriber works as well as a render prop returning `subscribers: string[]` so you can have multiple and custom subscribers. Correct if I'm wrong but seems the only way to have it.

**Comment by enricop89** [comment]
Hi @pedrobslisboa, thanks for filling the issue. I think the problem is with this [line](https://github.com/opentok/opentok-react-native/blob/main/%40types/index.d.ts#L423). 

It should be 

```
export class OTSubscriberView extends React.Component<ViewProps> {
    /**
     * OpenTok Subscriber streamId.
     */
    streamId?: string;
  }
```

Can you give it a try as well? IF your feedback is positive, I will create a PR.

**Comment by pedrobslisboa** [comment]
@enricop89 I've created an pull request about it https://github.com/opentok/opentok-react-native/pull/505

We can work on it 😁

**Comment by enricop89** [comment]
Solved with `0.17.1`. Thanks for the PR @pedrobslisboa !
